src/util/truncate.ts

```diff
@@ -0,0 +1,4 @@
+// 80 is the max str length
+export function truncate(content: string) {
+  return content.length > 80 ? `${content.slice(0, 80)}...` : content;
```

Comment:
are you sure not content.slice(0, 77) to have 80 char with the 3 dots ?

Reply:
yeah bc solène QA comment showed 80 chars excluding the ...

Reply:
sounds good ser